test/test_dynamic_shapes.py

```diff
@@ -33,13 +34,19 @@ def test_sizeAdd(self):
     t1[3][0] = 1
     # t2 has size [<=10, 2]
     t2 = torch.nonzero(t1)
-    # Exercise SizeAdd::getStaticValue
+    # Create a SizeAdd IR node.
+    # t2.shape[1] generates a SizeConstant node.
     dyn_size = t2.shape[0] + t2.shape[1]
-    # Exercises SizeAdd::getDynamicValue
+    # Exercises SizeAdd::getDynamicValue.
     dynamic_size = int(dyn_size)
     self.assertEqual(dynamic_size, 3)
-    # Exercise SizeAdd::getStaticValue
+    # Exercise SizeAdd::getStaticValue.
     self.assertEqual(str(dyn_size), '<=12')
+    t3 = torch.ones(1, device=dev)
+    # Exercise SizeAdd::Lower.
+    t4 = t3.expand(dyn_size)
+    self.assertEqual(t4.size(0), 3)
+    print(torch_xla._XLAC._get_xla_tensors_text([t4]))
```

Comment:
I wasn't able to test SizeAdd::ToString() method. Let me try to look further.

Reply:
what's the output of `torch_xla._XLAC._get_xla_tensors_text([t4])` here?  we want to do a `self.assertIn` for the IR here.

Reply:
The output of `torch_xla._XLAC._get_xla_tensors_text([t4])` here is
```
IR {
  %0 = s32[] prim::Constant(), value=2
  %1 = f32[] prim::Constant(), value=1
  %2 = f32[1]{0} aten::view(%1), output_size=(1)
  %3 = f32[] prim::Constant(), value=0
  %4 = f32[5,2]{1,0} aten::expand(%3), size=(5, 2)
  %5 = f32[1,2]{1,0} xla::generic_slice(%4), base_indices=(3, 0), sizes=(1, 2)
  %6 = f32[2]{0} aten::view(%5), output_size=(2)
  %7 = f32[2]{0} xla::update_slice(%6, %2), base_indices=(0)
  %8 = f32[1,2]{1,0} aten::view(%7), output_size=(1, 2)
  %9 = f32[5,2]{1,0} xla::update_slice(%4, %8), base_indices=(3, 0)
  %10 = (s32[<=10,2]{1,0}, s32[]) aten::nonzero(%9), num_outputs=2
  %11 = s32[] aten::size(%10.0)
  %12 = s64[] aten::add(%11, %0)
  %13 = f32[] prim::Constant(), value=1
  %14 = f32[1]{0} aten::expand(%13), size=(1)
  %15 = f32[<=12]{0} aten::expand(%14, %12), size=(12), dynamic_dims=(1), ROOT=0
}
```
It seems hard to test in the python code. But how about let me test in https://github.com/pytorch/xla/blob/1685a28821c6a00350bac39dd93faa6797c3c297/test/cpp/test_ir.cpp#L165

Reply:
You can just assert if `f32[<=12]{0} aten::expand` is in the `torch_xla._XLAC._get_xla_tensors_text([t4])`

Reply:
What I am trying to test is `SizeAdd::ToString()` instead of `expand`. I set `SizeAdd::String()` to return "SizeAdd for op aten::add" but as you can see the string does not appear in `torch_xla._XLAC._get_xla_tensors_text([t4])`.

So I don't think `torch_xla._XLAC._get_xla_tensors_text([t4])` will exercise `SizeAdd::ToString()`. That's why I suggest to test in `test/cpp/test_ir.cpp` in this pr.